intermediate-week-1/error/errorHandler.js

```diff
@@ -17,4 +17,5 @@ module.exports = (err, req, res, next) => {
   }
   logger.error(err.message);
   res.status(err.statusCode).json(data);
+  next()
```

Comment:
I'm not sure this is required. As if for any reason we get to the error handler there won't be any need to call next.